### DISCUSSION TITLE
Levelized Full System Costs of Electricity

### DISCUSSION THREAD

**Discussion by softwareengineerprogrammer** [NatLabRockies/GEOPHIRES-X#85] [Ideas]
https://www.sciencedirect.com/science/article/abs/pii/S0360544222018035?via%3Dihub

@malcolm-dsider I believe you mentioned integrating LFSCOE, perhaps this paper would be a good reference

**Comment by malcolm-dsider** [comment]
Yep that's the one I'm looking at. I haven't finished reading it, but technically, he's a fellow faculty member at Rice univ, so I'll reach out to him.


________________________________
From: Jonathan Pezzino ***@***.***>
Sent: Friday, December 22, 2023 9:18:48 PM
To: NREL/GEOPHIRES-X ***@***.***>
Cc: Malcolm Ross ***@***.***>; Mention ***@***.***>
Subject: [NREL/GEOPHIRES-X] Levelized Full System Costs of Electricity (Discussion #85)


https://www.sciencedirect.com/science/article/abs/pii/S0360544222018035?via%3Dihub

@malcolm-dsider<https://github.com/malcolm-dsider> I believe you mentioned integrating LFSCOE, perhaps this paper would be a good reference

**Comment by malcolm-dsider** [comment]
I have done further research on this and recommend that we DO NOT implement this. The first problem is that its inputs vary highly on a regional basis, so we would have to build an interface that selects the right data based on the project location. The idea of having default inputs based on regional location is a good one, but it requires a great deal of backend code development, and if we did that, I think there is lower hanging fruit to be enabled by this functionality (e.g., access geothermal gradient based on SMU data) than using it to calculate an obscure and potentially fundamentally flawed measure of LCOE. So we need to continue our search for a better comparable measure of LCOE. LFSCOE is not it.
